live-build/includes.chroot/usr/local/bin/record-system-info.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# In an effort to make troubleshooting system/boot issues
+# without granting console access possible, this script captures
+# information about the system state in a log file. 
+# TODOS?
+# if it looks like our Vx-vg var_encrypted volume is present
+# decrypt it and get more info?
+#
+# if Vx--vg-root is available, read-only mount and get more info?
+#
+# add tools to vx-iso for things like lsusb, lspci, etc...?
+#
+
+trap '' SIGINT SIGTSTP SIGTERM
+
+log_file="/tmp/system-state.log"
```

Comment:
small amount of paranoia – maybe generate a random filename?

Reply:
Sure, but could you elaborate on what the concern is?